python/ray/tune/scripts.py

```diff
@@ -0,0 +1,139 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+
+import click
+import glob
+import json
+import os
+from datetime import datetime
+
+import pandas as pd
+from ray.tune.trial import Trial
+from tabulate import tabulate
+
+
+def _flatten_dict(dt):
+    while any(type(v) is dict for v in dt.values()):
+        remove = []
+        add = {}
+        for key, value in dt.items():
+            if type(value) is dict:
+                for subkey, v in value.items():
+                    add[":".join([key, subkey])] = v
+                remove.append(key)
+        dt.update(add)
+        for k in remove:
+            del dt[k]
+    return dt
+
+
+@click.group()
+def cli():
+    pass
+
+
+DEFAULT_EXPERIMENT_INFO_KEYS = ("trial_name", "trial_id", "status",
+                                "num_failures", "logdir")
+
```

Comment:
DEFAULT_EXPERIMENT_INFO_KEYS = (
    "trial_name", 
    "trial_id",
    "status",
    "num_failures", 
    "logdir"
)

Reply:
oh I think this format is right (according to `yapf` and `scripts/format.sh`)